### ISSUE TITLE
ערוץ 12 עושה בעיות בכל הפלטפורמות 

### ISSUE THREAD

**Issue by taxi4u** [Fishenzon/repo#214]
כבר מעל חודש ערוץ 12 או שנפתח מאד איטי או שלא עולה בכלל וצריך ללחוץ על הלינק כמה פעמים .שמתי לב ש12 ליקויי שמיעה נפתח יותר טוב .אולי אפשר למצוא פיתרון ? גרסה 3.6.2 התקנה נקיה 
על קודי 21.3

**Comment by rafi-d** [comment]
סתם רעיון; אם במקרה עדכנת מגרסה קודמת של התוסף, למעשה לו disable ו enable , או הסבר הנוהתקנה.

**Comment by rafi-d** [comment]
לא יודע מאיפה אתה יודע לגבי "כולם", אני לא נתקל בבעיה כזו, אבל זה על קודי 19.5 ו 20.5 . אולי יש משהו ספציפי שקשור בגרסה 21... ה"מדיניות" שלי - לשדרג את קודי רק לגרסה לפני אחרונה (STABLE)....

אם אתה כבר מנסה ומשקיע בלהתקין מחדש, אולי תנסה על גרסה 20...
https://mirrors.kodi.tv/releases/

**Comment by taxi4u** [comment]
שמע רפי ,לגבי גרסה 19.5 אכן הוא עולה שם גם על בילד הפודל וגם על קמיליון אבל
שים לב לעליה שלו לוקח לו יותר מדי זמן להיפתח .ןאני מדבר איתך גם על סטרימרים
וגם על ווינדוס שלא לדבר על Raspberry pi שאני כותב כולם זה אחרי בדיקה עם
עשרות רבות של משתמשים החווים בעיות דומות .אתה צודק  לגבי חזרה לגרסאות ישנות
של עידן פלוס פעם הטריק הזה היה עובד אבל מקודי 21 זה כבר לא הולך .הסיבה
למעבר לקודי 21 פלוס נעוצה לאו דווקא בעידן פלוס אלא במבחר הסרטים והסדרות
וריבוי המקורות העובדים מעולה בגרסאות קודי החדשות ולא ניתן להתקין אותן בקודי
הישן .בעבר הבעיה היתה תמיד בערוץ 11 .מספר ימים מועט חווינו בעיה בערוץ 14
שמיד סודרה  ע"י פישנזון בגרסה 3,5,8. הבעיה הנוכחית קיימת בכל הפלטפורמות
ווינדוס ,אנדרואיד ,ורספברי פיי בחלק מהמכשירים ערוץ 12 בכלל לא עולה ונדרש
מספר פעמים  לנסות עד שהוא עולה .בחלק. הוא עולה אך לוקח לו זמן .כמובן
שווינדוס הוא עולה קצת יותר מהר .ככה לפחות המידע שאני מקבל יום יום . מצאתי
גם שערוץ 12. ליקויי שמיעה עולה טוב יותר אך עדיין קיימת איטיות בעליה .אני
מציף את הבעיה בעיקר כדי למצוא פיתרון .

בתאריך יום ו׳, 23 באוג׳ 2024, 17:51, מאת rafi-d ***@***.***>:

**Comment by rafi-d** [comment]
תמיד יותר מהר זה יותר טוב, אין וויכוח... 
לא עולה בכלל זו בעיה נפרדת, ומהר זה בעיה נפרדת... אז תפריד. אם ב20/19.5 זה תמיד מתנגן בסוף וב21 שזה לא - זה באג.

לגבי במהירות: זה פרמטר שלא קשור לתוסף, אלא לתקשורת וכוח CPU. אני אם חיבור קווי של 1G גם בסטרימרים (2X) וגם לאינטרנט. CPU S905X4 . לא מכיר את החיבוריות/תקשורת והCPU שלך , אבל אם בחלונות זה מהר יותר (אצלי זה כ-4 שניות) זה ההסבר...

**Comment by taxi4u** [comment]
רפי ידידי, הסיבה שאני מציין רק את ערוץ 12 . כי ערוצים 11-13-14-24 והשאר
נפתחים באמת מהר תוך
 3-4 שניות  ואילו ערוץ 12 לוקח לו כ 10שניות פלוס וזה התחיל רק לפני כחודש
דרך אגב הבעיה רק בשידור חי ב vod  דווקא עולה טוב בכל הפלטפורמות .מסכים איתך
שמי שחווה יותר איטיות או אי פתיחת הערוץ אלא לאחר מספר ניסיונות מהירות
האינטרנט  ומפרט חומרת  המכשיר אצלו משפיעה. עדיין זאת בעיה שלא היתה ולא
קיימת בשאר ערוצי עידן פלוס .ועל זה אני מצביע .אני מניח שבקרוב ימצא פיתרון
.לי אישית אין בעיה גם אם צריך להמתין עוד כמה שניות

בתאריך שבת, 24 באוג׳ 2024, 7:59, מאת rafi-d ***@***.***>:

**Comment by rafi-d** [comment]
נקווה ... אגב, יש עוד עניין קטן שלא עובד רק ב-12, אולי יש איזשהו קשר... #215

**Comment by rafi-d** [comment]
I agree with the *possible* scenario, just that it can easily be simulated to verify, by intentionally limiting your own connection speed (PC/router). So, I did and it is NOT the case:
https://drive.google.com/file/d/1H-0xPgKUxmDm-Mz4-JZfIA0dHgA15ifJ/view?usp=sharing
Startup time is ~3 seconds without speed limit, and ~25 seconds WITH speed limit to 3000Kbps (3Mbps). In both cases it is successfully being started, of cause buffering a lot after... ch 12 requires ~600KB/sec on average to avoid buffering (5Mbps).  FYI.

**Comment by tzagim** [comment]
#217
